sunpy/net/helio/parser.py

```diff
@@ -175,9 +181,10 @@ def link_test(link):
         None
     """
     try:
-        with closing(urllib.request.urlopen(link, timeout=LINK_TIMEOUT)) as fd:
```

Comment:
what's closing do?

Reply:
I assume it was before a time the urlopen was unable to be opened as a context manager.

Reply:
huuuh. mybbe.

Reply:
https://docs.python.org/3/library/contextlib.html#contextlib.closing